api/controllers/EventController.js

```diff
@@ -88,6 +95,27 @@ var EventController = {
         })     
     },
 
+    getNearestEvents: function(req, res) {
+        var lng = req.swagger.params.lng.value;
+        var lat = req.swagger.params.lat.value;
+        var radius = req.swagger.params.radius.value;
+
+        Event.find({ 'hasFinished': false }, function (e, events){
```

Comment:
Having all events in memory might cause performance issue. Mongodb features a `$near` operator you can take a look. The idea is let mongo do as much as it could